.github/workflows/ves-data-sync.yml

```diff
@@ -0,0 +1,104 @@
+name: VES Data Sync
```

Comment:
Can you provide insight as to why this was added to a workflow instead of a rake task or job?

Reply:
Similarly to the Income Limits data sync (https://github.com/department-of-veterans-affairs/vets-api/blob/master/.github/scripts/income-limits-data-sync.rb) it comes down to Oracle DB access. Oracle Instant Client doesn't have an Apple Silicon build, so we can't add the gem for general use to the project since it won't work on most of our machines. Building it as a Workflow lets us spin it up in an isolated container, running the Oracle DB query and saving it to CSVs that we push to S3. (We don't have access to the Rails ecosystem here, hence the separate Sidekiq job to import those CSVs - again, following the pattern for Income Limits: https://github.com/department-of-veterans-affairs/vets-api/blob/18b5b6bf4c802f3d5c2df4267b13c563b47a02cd/app/sidekiq/income_limits/std_income_threshold_import.rb#L8 .)

Is there a better alternative?